services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -63,7 +65,7 @@
 
     @JsonProperty(RegistryManagementConstants.FIELD_STATUS)
     @JsonInclude(value = Include.NON_EMPTY)
-    private Status status = new Status();
+    private Status status;
```

Comment:
I think it may good to avoid `null` cases here. Having an empty object object is better IMHO than `null`.

Reply:
The reason why I removed this is that `@JsonInclude(value = Include.NON_EMPTY)` does only work with null values for `Status` objects (unless we would write a custom serializer).

Since I wanted the status property not to show up at all in the JSON responses instead of an empty status property as below
```
   ...
   "status": {}
   ...
```
I decided to remove that line.